src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': tab.id === selectedTab.id })}
```

Comment:
```suggestion
              className={cn({ 'is-active': tab.id === selectedTab.id })}
```

Let's define constant for this like `isSelectedTab` and reuse in all places